src/pages/stage-page/validate-stage-page.ts

```diff
@@ -23,41 +23,22 @@ export const validateTimetableStage = ({
 
   const errors: ValidationErrorItem[] = [];
 
-  const startEventValidation = eventSchema.validate(
+  const stageValidation = stageSchema.validate(
     {
-      eventDate: startEvent.eventDate,
+      startDate: startEvent.eventDate,
+      endDate: endEvent?.eventDate,
       notes: startEvent.notes,
     },
     { abortEarly: false }
   );
 
-  if (startEventValidation.error) {
+  if (stageValidation.error) {
     errors.push(
-      ...startEventValidation.error.details.map((error) => ({
+      ...stageValidation.error.details.map((error) => ({
         ...error,
-        path: [startEventKey, ...error.path],
+        path: [formProps.title.replace(" ","-"), ...error.path],
```

Comment:
I believe that this will only replace the first " " with "-" to replace all " "s I think u need to use a regex

Reply:
Or `replaceAll` should work (but I'm not sure the browser support is as good)